### ISSUE TITLE
longdesc doesn't work as shown in the code for non-screen reader users

### ISSUE THREAD

**Issue by WayneEDick** [w3c/wai-tutorials#233]
Issue with Complex Images

There is still a problem with complex images: Users with partial sight who do not use screen readers, the vast majority, still cannot access longdesc.  The  problem is that there is no visual indicator of the presence of a longdesc and no visually accessible way to invoke a longdesc.

If browser support does not exist currently, then authors must code in access.  Author's do not need to be given browser support to make longdesc accessible to visual users  who still need longdesc.  The tutorial should teach authors how to do that.
For example: One could always force a tab stop on images with longdesc.  That is if the image element is not already tab-able always use a tabindex=0.  Then an on focus message could appear using css generated text to identify the presence of a longdesc.  Finally the image, in absence of a functional longdesc, should provide a direct link to the longdesc code. This could of course be coded to appear only if the image is tabbed to.

This is just one strategy.  Not necessarily the best, but anything is better than nothing.
Look at: Image Issue to see the problem.

Lest you think that screen magnification can really work, let us compare a complex image with 400%  to a normal 100% enlargement. This is a modest  enlargement for people with low vision:

http://nosetothepage.org/EO/CREB100.jpg and 
http://nosetothepage.org/EO/CREB400.jpg

This image is very small so it will fit in most window spaces,  but look at the degradation of print quality.  A smart screen magnifier cannot smooth text as an image.  Consider this same picture twice as large, but with no change in font size, a realistic scenario.  Imagine fighting the blurred text along with mouse driven scrolling.  Can you really claim that that is equally effective access?
This is a very realistic example of day-to-day issues faced by scholarly and professional people with partial sight. 
One more example: In the Roles Model Document there is an illustration of the Roles Taxonomy.  I have captured screen shots of the SVG version of this image at 80% and 170%.  The text is very readable in both, but navigating the larger image will be almost impossible. This access is not equally effective.

http://nosetothepage.org/EO/RoleTax80.jpg
http://nosetothepage.org/EO/RoleTax170.jpg

Note: I  chose 80% because it fit on one screen.  I chose 170% because it was the closest enlargement step I had that was near 200% of 80%. The Role Model page  does it right there is a link for everyone, including a link to the text description.

**Comment by yatil** [comment]
Hi Wayne,

please have a look at the current version of the Complex Images and see if your concerns are addressed properly. I think we have found a good balance:

https://w3c.github.io/wai-tutorials/images/complex/

**Comment by yatil** [comment]
This was resolved in EO discussions.

/me is doing some cleanup :-)
